src/main/kotlin/blackjack/model/FinalResult.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.model
+
+import blackjack.controller.Controller.Companion.BLACKJACK_SCORE
+
+class FinalResult(val dealer: Dealer, players: List<Player>) {
+    var win: List<Player> = listOf()
+        private set
+    var lose: List<Player> = listOf()
+        private set
+    var draw: List<Player> = listOf()
+        private set
```

Comment:
```suggestion
    val win: List<Player>
    val lose: List<Player>
    val draw: List<Player>
```

What do you think are the pros and cons of these two implementations?